test/esinstall/package-entrypoints/package-entrypoints-export-map.test.js

```diff
@@ -42,6 +42,23 @@ describe('package-entrypoints exports configuration', () => {
     }
   });
 
+  it('export-map-internal-imports', async () => {
+    const cwd = __dirname;
+    const dest = path.join(cwd, 'test-export-map-internal-imports');
+
+    console.log(await runTest(
```

Comment:
Missing some assertions on what this returns.

Reply:
doh, good catch! Was just testing that it didn't throw/fail